django-stubs/contrib/auth/backends.pyi

```diff
@@ -13,7 +13,7 @@ UserModel: Any
 
 class BaseBackend:
     def authenticate(self, request: HttpRequest | None, **kwargs: Any) -> AbstractBaseUser | None: ...
-    def get_user(self, user_id: int) -> AbstractBaseUser | None: ...
+    def get_user(self, user_id: Any) -> AbstractBaseUser | None: ...
```

Comment:
Personally I would have preferred `int | str | UUID`. But OK, let's keep this for now